src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.1.Wasm/MyExtensionsApp.1.Wasm.csproj

```diff
@@ -29,6 +29,10 @@
 		<!-- Temporarily uncomment to generate an AOT profile https://aka.platform.uno/wasm-aot-profile -->
 		<!-- <WasmShellGenerateAOTProfile>true</WasmShellGenerateAOTProfile> -->
 	</PropertyGroup>
+	<PropertyGroup Condition="'$(Configuration)'=='Debug' or '$(IsUiAutomationMappingEnabled)'=='True'">
+		<IsUiAutomationMappingEnabled>True</IsUiAutomationMappingEnabled>
+		<DefineConstants>$(DefineConstants);USE_UITESTS</DefineConstants>
+	</PropertyGroup>
```

Comment:
@dansiegel we probably should be enable this only when UI Tests are enabled?

Reply:
done